torch/quantization/_quantize_script.py

```diff
@@ -82,13 +92,23 @@ def get_scripted_qconfig_dict(qconfig_dict):
 
 def _prepare_script(model, qconfig_dict, is_dynamic):
     _check_is_script_module(model)
+    if not all(map(lambda x : isinstance(x, str), qconfig_dict.keys())):
+        raise ValueError('qconfig_dict should contain names(str) as keys. '
+                         'Run preprocess_qconfig_dict first to convert types to names')
     scripted_qconfig_dict = get_scripted_qconfig_dict(qconfig_dict)
     return wrap_cpp_module(torch._C._jit_pass_insert_observers(model._c,
                                                                'forward',
                                                                scripted_qconfig_dict,
                                                                False,
                                                                is_dynamic))
 
+def preprocess_qconfig_dict(module, qconfig_dict):
```

Comment:
Does this need to be a top level API? Can this be called inside prepare_script?

Reply:
We need to pass in the module here to be able to access child module. In prepare script we take scripted module so we lose child information

Reply:
the child info is still there, we lose the type info in scripted module

Reply:
same for this function, qconfig_dict should be the first argument.